.github/workflows/tests.yml

```diff
@@ -79,6 +79,10 @@ jobs:
         ./setup.sh
         ./setup-dev.sh
         mkdir .cov
+  
+    - name: Run cross-architecture tests
+      run: |
+        ./tests.sh -t cross-arch
```

Comment:
I moved these lines above the qemu system tests because they execute significantly faster - allows faster feedback for them. I noticed that if these fail, then the subsequent kernel testing doesn't take place. Are we okay with that?